### PR TITLE
karmadactl:print long describe by templates like kubectl

### PR DESCRIPTION
**Pull request by Fish-pro**
**What type of PR is this?**

/kind cleanup

**What this PR does / why we need it**:
The existing uses of short and long, include const, string, var, without uniform formatting rules, the code is hard to read,now fix it

**Which issue(s) this PR fixes**:
Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

File: pkg/karmadactl/get.go [hunk 1]
```diff
@@ -58,6 +58,7 @@ var (
 	eventColumn = metav1.TableColumnDefinition{Name: "EVENT", Type: "string", Format: "", Priority: 0}
 
 	getShort = `Display one or many resources`
+	getLog   = `Display one or many resources.`
```
Comment (lonelyCZ) [inline pkg/karmadactl/get.go:61]: `getLog` --> `getLong`

**Comment by prodanlabs** [comment]
Part of the job seems to conflict with #2163

File: pkg/karmadactl/get.go (hunk 1 above)
Comment (Fish-pro) [reply pkg/karmadactl/get.go]: @lonelyCZ Thank you for pointing out the error, it has been modified

**Comment by Fish-pro** [comment]
> #2163

@prodanlabs @lonelyCZ Oh, I just see that，this pr changes are more than #2163, I will deal with the conflict after #2163 merge

**Comment by lonelyCZ** [comment]
Can you describe the advantages of introducing `templates.LongDesc`？

**Comment by Fish-pro** [comment]
> Can you describe the advantages of introducing `templates.LongDesc`？

@lonelyCZ I found that some modules already introduced templates.LongDesc, so this pr holds all the short long formats together. I understand that as Karmadactl gets better, longer command descriptions are necessary. In the meantime, I think Karmadactl should have its own format for subcommands in the future, what's your opinion?
for example:
```go
// Long command description and examples.
var (
  mineLong = templates.LongDesc(`
    mine which is described here
    with lots of details.`)

  mineExample = templates.Examples(`
    # Run my command's first action
    %[1]s mine first_action

    # Run my command's second action on latest stuff
    %[1]s mine second_action --flag`)
)

// MineOptions contains all the options for running the mine cli command.
type MineOptions struct {
  mineLatest bool

  genericclioptions.IOStreams
}

// NewCmdMine implements the kubectl mine command.
func NewCmdMine(karmadaConfig KarmadaConfig, parentCommand string, streams genericclioptions.IOStreams) *cobra.Command {
  opts := &MineOptions{
    IOStreams: streams,
  }

  cmd := &cobra.Command{
    Use:     "mine",
    Short:   "Run my command",
    Long:    mineLong,
    Example: fmt.Sprintf(mineExample, parentCommand),
    Run: func(cmd *cobra.Command, args []string) {
      if err := opts.Complete(karmadaConfig, cmd, args); err != nil {
        return err
      }
      if err := opts.Validate(cmd, args); err != nil {
        return err
      }
      if err := opts.Run(); err != nil {
        return err
      }
    },
  }

  cmd.Flags().BoolVar(&options.mineLatest, "latest", false, "Use latest stuff")
  return cmd
}

// Complete completes all the required options for mine.
func (o *MineOptions) Complete(f *cmdutil.Factory, cmd *cobra.Command, args []string, out io.Writer) error {
  return nil
}

// Validate validates all the required options for mine.
func (o MineOptions) Validate() error {
  return nil
}

// Run implements all the necessary functionality for mine.
func (o MineOptions) Run() error {
  return nil
}
```

**Comment by lonelyCZ** [comment]
Because there are some commands that have a short `Long describe`, introducing `templates.LongDesc` seems not necessary. I tend to use it when we have a real Long description.

**Comment by Fish-pro** [comment]
> Because there are some commands that have a short `Long describe`, introducing `templates.LongDesc` seems not necessary. I tend to use it when we have a real Long description.

@lonelyCZ Ok, so now short descriptions, some are defined by var,some passed in as string arguments, does that need to be uniform?

**Pull request closed without merging**